src/renderer/containers/AddValidator/CheckPassword.tsx

```diff
@@ -38,27 +34,20 @@ Pick<IRootState, "register" | "addValidator" | "auth">> = (props) => {
         if(accounts!=null){
             const isCorrectValue = await accounts.isCorrectPassword(input);
             if(isCorrectValue) {
-                
+
                 const signingKey = PrivateKey.fromBytes(
-                    Buffer.from(props.register.signingKey.slice(2), "hex")
-                );
-                
-                if(props.auth.auth !== null) {
+                    Buffer.from(props.register.signingKey.slice(2), "hex"));
```

Comment:
it's risky to assume it will always have prefix "0x". Safer option would be .replace("0x", "")